src/Query/FetchMappedProductsQuery.php

```diff
@@ -30,7 +30,8 @@ public function fetch(string $catalogId): array
 
         $products = [];
         foreach ($rawMappedProducts as $rawMappedProduct) {
-            $label = $rawMappedProduct['title'] ?: $rawMappedProduct['uuid'];
+            $label = $rawMappedProduct['title'] ?? $rawMappedProduct['uuid'];
+            $label = '' === $label ? $rawMappedProduct['uuid'] : $label;
```

Comment:
duplicate replacement I think

Reply:
Not really: the first is for a missing key the second is for an empty string (initially I used one of the 2 and discovered a bug)

Reply:
@RiffVentura : suggestion : `$rawMappedProduct['title'] == '' ? $rawMappedProduct['uuid'];`

Reply:
This would fail as the 'title' might be undefined
The alternative would be that
```php
$label = isset($rawMappedProduct['title']) && $rawMappedProduct['title'] !== ' '
? $rawMappedProduct['title'] 
: $rawMappedProduct['uuid'];
```
Would you prefer this?

Reply:
Oh yes, gg.

But more simple : 
`$label = $rawMappedProduct['title'] ? $rawMappedProduct['title'] : $rawMappedProduct['uuid'];`
Reading [this documentation](https://developer.mozilla.org/en-US/docs/Glossary/Truthy), it will do what you want.

Reply:
This would fail as the 'title' might be undefined (it generates a warning) and the documentation link is about JS

Reply:
Ok sorry for this useless debate @RiffVentura . IMO I prefer your last suggestion. It's not very beautiful but more explicit than the first on.